app/views/groups/group_links/_invited_groups.html.erb

```diff
@@ -0,0 +1,38 @@
+<%= turbo_frame_tag('members') do %>
+  <div class="flex flex-col">
+    <div class="overflow-x-auto">
+      <div class="shadow">
+        <table class="min-w-full table-fixed dark:divide-gray-600">
+          <thead class="text-gray-700 bg-gray-50 dark:bg-gray-700 dark:text-gray-400">
+            <tr>
+              <th scope="col" class="px-6 py-3 text-left"><%= t(:"groups.group_links.index.table_header.group") %></th>
+              <th scope="col" class="px-6 py-3 text-left"><%= t(:"groups.group_links.index.table_header.source") %></th>
+              <th scope="col" class="px-6 py-3 text-left"><%= t(:"groups.group_links.index.table_header.access_granted") %></th>
+              <th scope="col" class="px-6 py-3 text-left"><%= t(:"groups.group_links.index.table_header.access_level") %></th>
+              <th scope="col" class="px-6 py-3 text-left"><%= t(:"groups.group_links.index.table_header.expiration") %></th>
+              <th scope="col" class="px-6 py-3 text-left"><%= t(:"groups.group_links.index.table_header.action") %></th>
```

Comment:
**nitpick:** Can you align the 'Action' header and the column contents to the far right of the column

Reply:
Definitely looks better, thanks